src/libs/ReportUtils.ts

```diff
@@ -1930,7 +1930,7 @@ function getPolicyExpenseChatName(report: OnyxEntry<Report>, policy: OnyxEntry<P
      * Using typical string concatenation here due to performance issues
      * with template literals.
      */
-    const policyItem = allPolicies?.[ONYXKEYS.COLLECTION.POLICY + report?.policyID];
+    const policyItem = allPolicies?.[`${ONYXKEYS.COLLECTION.POLICY}${report?.policyID}`];
```

Comment:
Should we be concerned about the performance impact mentioned in the comment? Otherwise I guess the comment should be updated.

Reply:
Reading through google i thing string literals (`${}`) is now faster, have a look [here](https://stackoverflow.com/a/29083467/21869664).

I think we should just remove the comment, wdyt?

Reply:
Changed [here](https://github.com/Expensify/App/pull/36849/commits/7907c17604578108e68802b21a4545bb49e513f9)